oneanddone/tasks/templates/tasks/activity_listing.html

```diff
@@ -24,6 +24,12 @@
           <div>{{ _("Team: {team}")|fe(team=attempt.task.team.name) }}</div>
           <div>{{ _('Date {state}: {modified}')|fe(state=attempt.get_state_display(),
             modified=attempt.modified.strftime('%Y-%m-%d: %I:%M:%S %p').lower()) }}</div>
+          <div>
```

Comment:
I suggest moving the `<div>` inside the `{% if` so that if we do not have a `time_spent_in_minutes` we do not even output the `<div>` to the page.